src/test/scala/sigmastate/serialization/DeserializationResilience.scala

```diff
@@ -12,6 +12,11 @@ class DeserializationResilience extends PropSpec
     an[ArrayIndexOutOfBoundsException] should be thrownBy ValueSerializer.deserialize(Array[Byte]())
   }
 
+  property("max size limit") {
+    val size = 1024 * 1024 * 10
```

Comment:
@ergomorphic should we assume that it's being taken care of by tx max size limit and don't bother checking?

Reply:
@greenhat I think we should check everything we can.